### PR TITLE
create moyo header

### PR DESCRIPTION
**Pull request by Anothar**
- [DEMO LINK](https://anothar.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://anothar.github.io/layout_moyo-header/report/html_report/)

- [x] Header height is set in 1 place (for the links)
- [x] Content is vertically centered (for any header height)
- [x] CSS is used to show all letters in Uppercase (don't type them in HTML)
- [x] Logo is an image wrapped with a link
- [x] CSS Variable is used for a blue color
- [x] Pseudo-element is used for a blue line below the active link
- [x] Code follows all the [Code Style Rules ❗️](https://github.com/Anothar/layout_moyo-header/blob/master/checklist.md)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,49 @@
...
+            <a class="nav__link" href="/">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="/">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link"
+              href="/"
+              data-qa="hover"
+            >Laptops & Computers</a>
```
Comment (Viktor-Kost) [inline src/index.html:37]: code styling
```suggestion
            <a 
              class="nav__link"
              href="/"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  padding: 0;
```
Comment (Viktor-Kost) [inline src/style.css:3]: body doesn't have default padding
```suggestion
```

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,65 @@
...
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  list-style: none;
+}
+
+.nav__link {
+  display: inline-block;
+  line-height: 60px;
```
Comment (Viktor-Kost) [inline src/style.css:24]: set a correct value from figma

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,65 @@
...
+  height: 60px;
```
Comment (Viktor-Kost) [inline src/style.css:25]: it's not a good practice to hardcode height for elements with content inside. use padding for your links instead

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,65 @@
...
+  color: #000;
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
```
Comment (Viktor-Kost) [inline src/style.css:28]: move to `html` tag

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,65 @@
...
+  margin: 0 10px;
+}
+
+.nav__list .nav__item:last-child .nav__link {
+  margin-left: 10px;
+}
+
+.nav__list .nav__item:first-child .nav__link {
+  margin-right: 10px;
+}
```
Comment (Viktor-Kost) [inline src/style.css:49]:
```suggestion
.nav__item:not(:last-child) {
  margin-right: 20px;
}
```

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,65 @@
...
+  text-decoration: none;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+.nav__link:hover {
+  color: #00ACDC;
```
Comment (Viktor-Kost) [inline src/style.css:36]: move repeated colors to variables

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

**Comment by Anothar** [comment]
please check)

File: src/index.html [hunk 8]
```diff
@@ -7,8 +7,52 @@
...
+      <a href="/" class="logo__link">
+        <img class="logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="/">Apple</a>
```
Comment (DarkMistyRoom) [inline src/index.html:25]:
```suggestion
            <a class="nav__link is-active" href="#">Apple</a>
```

Comment (DarkMistyRoom) [reply src/index.html:25]: fix for all other links

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,71 @@
-body {
+* {
```
Comment (DarkMistyRoom) [inline src/style.css:1]: Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser
to read your web document

File: src/style.css [hunk 10]
```diff
@@ -1,3 +1,71 @@
...
+  justify-content: space-between;
+}
+
+.logo__link {
+  display: inline-block;
+  line-height: 0;
+}
+
+
+.nav__list {
```
Comment (DarkMistyRoom) [inline src/style.css:30]:
```suggestion
}

.nav__list {
```

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
- The test link doesn't work
- the logo link shouldn't lead to 404, as well as other ones
image

**Review by lerastarynets — CHANGES_REQUESTED** [review]
Test link still doesn't work and clicking on logo/tabs still leads to 404, pls fix

File: src/index.html [hunk 11]
```diff
@@ -7,8 +7,52 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo__link">
+        <img class="logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
```
Comment (anastasiiavorobiova) [inline src/index.html:20]:
```suggestion
        <img 
          class="logo"
          src="./images/logo.png"
          alt="logo"
        >
```

File: src/style.css [hunk 12]
```diff
@@ -1,3 +1,68 @@
+html {
+  --color-link-nav: #000;
+  --decoration-link-nav: none;
+  --font-link-nav: Roboto, sans-serif;
+  --active-color: #00ACDC;
+}
+
```
Comment (anastasiiavorobiova) [inline src/style.css:7]:
```suggestion
* {
  box-sizing: border-box;
}
```

File: src/style.css [hunk 13]
```diff
@@ -1,3 +1,68 @@
...
 body {
   margin: 0;
+  box-sizing: border-box;
```
Comment (anastasiiavorobiova) [inline src/style.css:10]:
```suggestion

```

**Review by anastasiiavorobiova — APPROVED** [review]
Great work! Added a few comments to fix

**Pull request closed without merging**